### PR TITLE
Release website under BSD license

### PR DESCRIPTION
**Pull request by jarrodmillman**
I couldn't find what license the website content and theme was released under.  If I missed it, please let me know.  If not, this PR adds a revised BSD license to the repo.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2020-2021, NumPy Developers
```
Comment (rgommers) [inline LICENSE:3]: Let's update this to 2010, that's the oldest commit in this repo (just not in the `master` branch). The new site started in 2019.

**Review by rgommers — COMMENTED** [review]
Thanks @jarrodmillman, good idea. This repo is indeed missing a separate license file. The website itself links to https://github.com/numpy/numpy/blob/master/LICENSE.txt (see `content/<language>/about.md`), can you change that to point to this new license file?

File: LICENSE (hunk 1 above)
Comment (jarrodmillman) [reply LICENSE]: Done.

**Comment by jarrodmillman** [comment]
I am happy to do whatever, but I want to double-check whether I should update `content/<language>/about.md`.  That seems to be a reference to the license for the numpy package and not numpy.org.  Are you sure I should update that to point to the numpy.org LICENSE file?

Here is an old discussion about this:
https://github.com/numpy/numpy.org/issues/138

**Comment by rgommers** [comment]
>  That seems to be a reference to the license for the numpy package and not numpy.org. Are you sure I should update that to point to the numpy.org LICENSE file?

Ah you are right! Okay then this is good to go.

**Review by rgommers — APPROVED** [review]

**Pull request merged**